rskj-core/src/main/java/co/rsk/peg/federation/FederationStorageProviderImpl.java

```diff
@@ -367,6 +375,15 @@ private void savePendingFederation(ActivationConfig.ForBlock activations) {
         bridgeStorageAccessor.saveToRepository(PENDING_FEDERATION_KEY.getKey(), serializedPendingFederation);
     }
 
+    private void saveProposedFederation(Federation proposedFederation) {
+        if (!isProposedFederationSet) {
```

Comment:
Should we also check for RSKIP activation? To make sure nothing is stored before the RSKIP activates

Reply:
i thought about that but i don't think that's needed, because before rskip `isProposedFederationSet` should be always false

Reply:
Done